packages/select/src/components/select/multiSelect.tsx

```diff
@@ -235,7 +233,7 @@ export class MultiSelect<T> extends React.PureComponent<IMultiSelectProps<T>, IM
             if (which === Keys.ESCAPE || which === Keys.TAB) {
                 // By default the escape key will not trigger a blur on the
                 // input element. It must be done explicitly.
-                this.input.blur();
+                this.input && this.input.blur();
```

Comment:
please don't do this type coercion for statements; it's confusing. we can be more lenient about inline expressions (but I personally still don't love it)